jenkins/jobs/kata-containers-crio-PR/config.xml

```diff
@@ -213,6 +213,14 @@ echo &quot;CRI-O Version to test:&quot;
 sudo crio --version
 crictl --version
 export RUNTIME=kata-runtime
+sudo -E PATH=$PATH make crio
+
+echo &quot;Ensure crio service is stopped before running the tests&quot;
+if systemctl is-active --quiet crio; then
+        sudo systemctl stop crio
+fi
+# Execute CRI-O tests using containerd-shim-kata-v2 runtime
+export RUNTIME=containerd-shim-kata-v2
```

Comment:
@fidencio not related with your change exactly...but using `RUNTIME` to switch between kata runtimes seems fragile. I explain:
1. In `tests/Makefile` it has:
```
crio:
        bash .ci/install_bats.sh
        RUNTIME=${RUNTIME} ./integration/cri-o/cri-o.sh
```
Forget for a moment that this `RUNTIME=${RUNTIME}` seems not needed.

2. Then in `integration/cri-o/cri-o.sh` it has exported `export CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-$RUNTIME}"`

4. IIUC cri-o test scripts use `CONTAINER_RUNTIME`

5. So if by mistake `CONTAINER_RUNTIME` gets in the env, then `RUNTIME` has no effect.